addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -27,8 +27,8 @@
 // GRID LAYOUT
```

Comment:
Note that I totally agree with you for both points, it clearly improves the visual :+1:  What I am saying is that this should not be introduced like this for the sake of consistency :wink:

TL,DR: let's drop this commit for now, as this is not really an "issue", just some current limitations :+1: We will try to improve the options for mobile in the future :+1: For the gutter, we could do it but it must be generalized to all containers, not only grid ones :wink: Let's merge without it first, we will have the time to discuss it again with QSM/LEBL as we are still in master and this is not a simple CSS decision to make :+1: 

---
- About the gaps, given that there is no option (yet ?) to set gaps on normal columns, they should not be allowed on mobile (yet) because it will be inconsistent with the other non-grid snippets (and it will break current websites) => once the option will be generalized, we will be able to use them :wink: So we will take it as a feedback and see if we can improve the options :wink: Given that the snippets have always looked like that, this is not really an issue if they are still like that for a little while more => I will create a task and we will see what we will do about that :+1:  

- About the gutter, what I said is that if you really think that a gutter should be set for some container size in mobile, then it should be done for **every** container, not only the grid mode ones, as it will be inconsistent otherwise :+1: So either generalize it like a `--gutter-x-mobile` and apply it in a general rule about containers, or do not do it at all :slightly_smiling_face: But I really think it should be discussed first :+1: (we still have time to do that after merging this PR without it :wink:)
Personally, I don't really think that not touching the viewport in desktop means also not touching in mobile :thinking: What if the user didn't want the space in mobile ?
For me, we should take this as a feedback in order to make more mobile-specific options later :slightly_smiling_face: 

I will create a task with these suggestions :+1: 

> Otherwise I'm not sure if we can merge "commit 2" neither, it will just expose the design issue even more

We can clearly merge commit 2, it was already like that for years so this is not an issue :wink: For me, the mobile view is clearly independent from the desktop view :+1: We already improved things to make mobile more customizable (order, sizing) so it is just a matter of time before the mobile view will truly be improved